apps/dataset-browser/src/messages/en/messages.json

```diff
@@ -54,6 +54,7 @@
     "back": "Back to datasets",
     "navigation": {
       "about": "About",
+      "measurements": "Transparency measurements",
```

Comment:
@GertjanFi Small update: I propose that we finalize this PR so that we can deploy it. You can then do a review on the live environment and give feedback (we'll probably have to change other texts related to the metrics and measurements, too, as their meaning may not be obvious to everyone).